.github/workflows/ci.yml

```diff
@@ -13,12 +13,13 @@ on:
 jobs:
   build-test:
     strategy:
+      # We want to know the status of all OTP versions, not just if one of them fails.
+      fail-fast: false
       matrix:
         versions:
         - {os: 'ubuntu-20.04', otp: '21.x', rebar: '3.15.2'}
-        - {os: 'ubuntu-22.04', otp: '24.x', rebar: '3.18.0'}
         - {os: 'ubuntu-22.04', otp: '25.x', rebar: '3.18.0'}
-        - {os: 'ubuntu-22.04', otp: '26.x', rebar: '3.22.0'}
+        - {os: 'ubuntu-22.04', otp: '27.x', rebar: '3.23.0'}
```

Comment:
OTP 27 just came out, so add it to the test matrix and drop a couple other versions.